rockchip: use USB host by default on rk3399-rock-pi-4

mj22226 opened the pull request:
This backport  fix connections errors
on the upper USB3 port of the Radxa ROCK Pi 4.

Link: https://lore.kernel.org/r/20201201154132.1286-4-vicencb@gmail.com

ynezz commented on target/linux/rockchip/patches-5.4/104-rockchip-use-USB-host-by-default-on-rk3399-rock-pi-4.patch:
```diff
@@ -0,0 +1,33 @@
+From 75b1c522cc33fa9b79077802a49803c874f647b1 Mon Sep 17 00:00:00 2001
+From: Marty Jones <mj8263788@gmail.com>
```
> I backported this patch from upstream to fix connections errors

Then please keep it as it is, now you've claimed yourself as the author of the patch which is not correct.

mj22226 replied:
I've corrected the mistake.

mj22226 committed:
rockchip: use USB host by default on rk3399-rock-pi-4

This backport  fix connections errors on the
upper USB3 port of the Radxa ROCK Pi 4 .

Signed-off-by: Marty Jones <mj8263788@gmail.com>

ynezz commented:
Thanks! Pulled into my staging tree at https://git.openwrt.org/openwrt/staging/ynezz.git

ynezz commented:
I've just noticed that your commit lacks `Signed-off-by:` so you need to fix it.

mj22226 commented:
> I've just noticed that your commit lacks `Signed-off-by:` so you need to fix it.

@ynezz Ok` Signed-off-by:` fixed

ynezz commented:
Thanks! Pulled into my staging tree at https://git.openwrt.org/openwrt/staging/ynezz.git
